readme.md

```diff
@@ -159,6 +159,16 @@
 | ------: | -------: | ---: |
 |       2 |        8 |  10m |
 
+### [Battleball](https://boardgamegeek.com/boardgame/6795/battleball)
+
+> Battleball is a football board game but with teams made of players with physical and mechanical enhancements - which has made the sport more intense and more deadly. Plan a strategy and do your best to score two touchdowns without having all of your players be destroyed!
+
+![Battleball Image](https://i.ebayimg.com/images/g/xFsAAOSwOMpd6G8M/s-l400.jpg)
```

Comment:
```suggestion
![Battleball Image](https://cf.geekdo-images.com/itemrep/img/_ybsCFVKunSnqyGPCiuXbT2kuDo=/fit-in/246x300/filters:strip_icc()/pic227687.jpg)
```

Like I mentioned in the first commit, there's a specified image size for the cdn path; finding the correct pre-generated image source means there's a more size appropriate version available to use.

Reply:
This looks good.